toolchain/build-xenon-toolchain

```diff
@@ -27,161 +35,180 @@ BUILD_GCC_SECOND=true
 EXTR=cat
 if type "pv" &> /dev/null; then
   EXTR=pv
-fi;
+fi
 
+cd "$(dirname "$0")"
 #FSBRANCH=Swizzy
 
 # path to the logfile
-LOGFILE="`pwd`/build.log"
+LOGFILE=build.log
 
-# temp variables export
-export DEVKITXENON="$PREFIX"
-export PATH="$PATH:$DEVKITXENON/bin:$DEVKITXENON/usr/bin"
+# temp variables export, same as xenon-env cmd does
+export DEVKITXENON=$PREFIX
```

Comment:
Even if spaces don't work with GCC as you stated in your comment, isn't it still a good idea to leave the quotes here for best practices? i mean, it could potentially break the entire script otherwise can't it?

Reply:
I have made this 'space safe' in every use of the variable.